### ISSUE TITLE
[Request]: Send an email only once when tagged series is imported

### ISSUE THREAD

**Issue by samip5** [Sonarr/Sonarr#3613]
**Describe the problem**
Not really a problem, but a nuance (if that's even a word). 
I would like to get only a summary of all the episodes that were imported and not one notification per each episode separately.

**Describe any solutions you think might work**
Some kind of email queue that would get all of the pending episode notifications and append it to one message after all of them has been imported for that day, this has to do with ongoing shows.

**Additional context**
N/A

**Comment by markus101** [comment]
This is already covered by https://github.com/Sonarr/Sonarr/issues/363
